api/proto/teleport/legacy/client/proto/authservice.proto

```diff
@@ -553,14 +553,21 @@ message Features {
   // IdentityGovernance indicates whether IGS related features are enabled:
   // access list, access request, access monitoring, device trust.
   bool IdentityGovernance = 23 [(gogoproto.jsontag) = "identity_governance,omitempty"];
-  // AccessGraph indicates whether Teleport Access Graph is enabled.
+  // AccessGraph enables the usage of access graph.
+  // NOTE: this is a legacy flag that is currently used to signal
+  // that Access Graph integration is *enabled* on a cluster.
+  // *Access* to the feature is gated on the `Policy` flag.
+  // TODO(justinas): remove this field once "TAG enabled" status is moved to a resource in the backend.
```

Comment:
Currently, Teleport Policy is a product that includes TAG feature. In the future, we may include more features in the Teleport Policy, and I think we should let people disable TAG feature anyways, as it's a separate service.

Reply:
We will definitely allow to turn TAG on and off. This TODO is more because `Features` is not a particularly good way to indicate it IMO*. Since we'll have a resource in the backend with TAG connection info to propagate from Auth to Proxy, this resource can also act as an "enabled" flag.

* To expand: traditionally `Features` indicated what the license supports, and not what is factually enabled on the cluster. Recent additions have muddied these waters a bit, but since we're introducing a proper license-based feature flag here, and we'll have the "access graph parameters" resource, we can undo this decision for at least TAG 🙂